sv-comp/my-bench-sv-comp/goblint-all-fast.sh

```diff
@@ -2,9 +2,9 @@
 
 shopt -s extglob
 
-MYBENCHDIR=/mnt/goblint-svcomp/benchexec/my-bench-sv-comp
-RESULTSDIR=/mnt/goblint-svcomp/benchexec/results/70-all-fast-no-interval
-GOBLINTPARALLEL=14
+MYBENCHDIR=//mnt/goblint-svcomp/sv-comp/goblint/sv-comp/my-bench-sv-comp
+RESULTSDIR=/mnt/goblint-svcomp/benchexec/results/LoopsEnums
+GOBLINTPARALLEL=3
```

Comment:
The changes to all-fast and data-race scripts and definitions should be reverted since they just reflect just some of your own experiments and nothing fundamental about these example scripts.
The no-overflow ones can remain since we didn't have any before.